odxtools/envdata.py

```diff
@@ -49,18 +51,15 @@ def from_et(et_element, doc_frags: List[OdxDocFragment]) -> "EnvironmentData":
             short_name=short_name,
             long_name=long_name,
             description=description,
+            sdgs=sdgs,
             is_visible_raw=is_visible_raw,
             parameters=parameters,
             byte_size=byte_size,
             dtc_values=dtc_values,
         )
 
     def _build_odxlinks(self) -> Dict[OdxLinkId, Any]:
-        odxlinks = super()._build_odxlinks()
-
-        odxlinks[self.odx_id] = self
-
-        return odxlinks
+        return super()._build_odxlinks()
```

Comment:
mainly because the class has a `_resolve_references()` method, so it's probably convenient to have it around. that said, in this case `_resolve_references()` also does basically nothing, so let's remove both. Thanks for the catch!